modules/hiding-information-about-applications.adoc

```diff
@@ -0,0 +1,40 @@
+:_module-type: PROCEDURE
+
+[id="hiding-information-about-applications_{context}"]
+= Hiding information about applications
+
+[role='_abstract']
+You can view a list of available applications in the *Exploring applications* page of the {productname-short} dashboard. You can optionally  disable the following extra information that is provided about the applications by default:
+
+* Any independent software vendor (ISV) application is indicated with a label on the tile indicating “Red Hat managed”, “Partner managed”, or “Self-managed”. 
+Optionally, as an admin user, you can hide the labels, for example, if you are running a self-managed environment and you want to show all available applications regardless of the support level. 
+
+* When a user clicks on an application, an information panel appears and provides more information about the application, including links to quick starts or detailed documentation. Optionally, you can disable the appearnce of application information panels.
+
+.Prerequisite
+
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+
+
+.Procedure
+
+. Log in to the {openshift-platform} console as a cluster administrator.
+. Open the dashboard configuration file:
+.. In the *Administrator* perspective, click *Home* -> *API Explorer*.
+.. In the search bar, enter `OdhDashboardConfig` to filter by kind.
+.. Click the `OdhDashboardConfig` custom resource (CR) to open the resource details page.
+.. Select the `redhat-ods-applications` project from the *Project* list.
+.. Click the *Instances* tab.
+.. Click the `odh-dashboard-config` instance to open the details page.
+.. Click the *YAML* tab. 
+
+. In the `spec:dashboardConfig` section, set either or both of the following options to `true` to disable the default behavior:
```

Comment:
Could probably cut 'either or both of' here.

Reply:
Leaving this one since they do not need to set them both and I'd like to make that clear